### ISSUE TITLE
[BUG]: Fluent Detail list blurriness

### ISSUE THREAD

**Issue by PeacockIADev** [microsoft/powercat-creator-kit#477]
**Describe the bug**
We have implemented the fluent detail list component in several of our production apps for almost a year now. We like the flexibility, ease of deployment and available features in the component. We currently have 3 almost identical apps all using the same component with the same theme style and sorting / filter behavior. 
However we noticed that the lists in one of the apps has become blurry.  

**To Reproduce**
1. We have implemented the list
2. Customized the app theme JSON to match the client company style and used the alternate row color feature.
3. We have limited the collection size to 500 to reduce loading and improve performance.
4. Used a custom sorting and filtering

**Expected behavior**
Since we have 3 almost identical apps, with identical AppThemeJSON, Alternate row colors and behavior we expect all lists to match. 
**Screenshots**
blurry
![image](https://github.com/microsoft/powercat-creator-kit/assets/160228733/c83d696a-0a37-4e81-b614-717cb3e9688f)
not blurry
![image](https://github.com/microsoft/powercat-creator-kit/assets/160228733/e2c4c5b7-bca1-4efe-b92a-727fde45cb8b)

**Additional context**
Both apps used to match. However since the power apps update regarding custom themes, one of them started to differ.

**Comment by PeacockIADev** [comment]
Hi Denise,

Thank you so much for your response!

To answer your questions, we see the blurriness on both the studio version as in the published application. This is present on every environment ( 4 in total).

***@***.*** ***@***.***
Both apps (with and without the blurriness) run on the same studio version, this is the latest version 3.24083. It has been present for a couple of months now, so the studio has been through multiple versions.

We were able to identify a difference between the 2 apps in published versions. When we open the app in the browser and inspect the html, we have noticed that there is a CSS rule “animation-name: css-155”  that is only present in the blurry app.   See screenshot below:

***@***.***
If we manual disable this rule in the inspector the blurriness disappears, however we don’t have access to the CSS from the studio.
We have no idea why this rule is there and how it has got there, but we would like to be able to disable it.

We are in contact with Microsoft for this issue, but no solution is given yet and they keep referring to you.

Hopefully, I have given you sufficient information regarding this issue, but if you have any questions, please let me know.
Hope to hear from you soon.


Kind regards,

**Comment by teapardee** [comment]
Hi @denise-msft @PeacockIADev , was this ever resolved? I am experiencing this issue as well. Thanks.

**Comment by kamalksharma** [comment]
Hello Everyone, do we have the work around or any fixes for this issue and am I too getting this issue. I have used Detailed List grid in 4 screen and out of those 3 screen are return proper output but only in 1 screen it is showin blurry data.

**Comment by PeacockIADev** [comment]
No, We are still having this issue. We have been through multiple support channels but no one seems to be able (or willing) to help :(

If someone finds a solution or fix please let me know!
